### PR TITLE
Update TWIF 18 - Add consume to end and connectors examples

### PR DESCRIPTION
**Pull request by tjtelan**
image

### CODE REVIEW

File: content/news/this-week-in-fluvio-0018.md [hunk 1]
```diff
@@ -61,9 +61,47 @@ For more detail on the available config options, see the [Rust docs](https://doc
...
+# 3rd-party-test-connector-config.yaml
+version: v1
+name: my-third-party-connector
+type: "https://raw.githubusercontent.com/infinyon/fluvio-connectors/main/rust-connectors/utils/test-connector/connector.yaml"
+topic: my-test-connector-topic
+create_topic: true
+direction: source
+```
+
+The YAML config at the URL only has one field `image`. The value is the docker hub image name.
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0018.md:90]: I don't see the `image` field in the `yaml` above.

File: content/news/this-week-in-fluvio-0018.md [hunk 2]
```diff
@@ -61,9 +61,47 @@ For more detail on the available config options, see the [Rust docs](https://doc
...
+
+%copy%
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0018.md:92]: Remove %copy% if this is for reference purpose only.

File: content/news/this-week-in-fluvio-0018.md [hunk 3]
```diff
@@ -85,6 +123,72 @@ $ fluvio version switch latest
...
+> 9
+Ok!
+> 10
+Ok!
+```
+
+Record indexing is 0-based, so we expect the stream to close when we receive the 6th record.
+
+```bash
+$ fluvio consume --end-offset 5 twif
```
Comment (ajhunyady) [inline content/news/this-week-in-fluvio-0018.md:164]: We should add `-B` to avoid confusion.

File: content/news/this-week-in-fluvio-0018.md (hunk 1 above)
Comment (tjtelan) [reply content/news/this-week-in-fluvio-0018.md]: It was in the code block below, but I've cleared this up

**Review by ajhunyady — APPROVED** [review]

**Pull request merged**